### PR TITLE
Add Chat with setup specialist to the accounting page.

### PR DESCRIPTION
**Pull request by nkdengineer**
### Fixed Issues

$ https://github.com/Expensify/App/issues/51487
PROPOSAL: https://github.com/Expensify/App/issues/51487#issuecomment-2446168063

### Tests

1. Login with a new email that is not alias email format(include +)
2. Create a new workspace
3. Enable Accounting
4. Go to the Accounting page
5. Verify that: Chat with setup specialist to the accounting page is displayed
- [x] Verify that no errors appear in the JS console

### Offline tests

same as above
### QA Steps

// TODO: These must be filled out, or the issue title must include "[No QA]."
same as above
- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/user-attachments/assets/c50848f6-cc97-4dd1-8782-cc264f33942c

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/user-attachments/assets/8fe93f65-dfdb-4b9a-b788-3c00c6afe436

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/user-attachments/assets/dca6e496-e8e1-4903-9c53-b52bb6c6ac6d

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/user-attachments/assets/a637dd58-e3b7-4780-bdc3-2db9f6c0c4ae

</details>

### CODE REVIEW

**Comment by nkdengineer** [comment]
I asked for translation into Spanish [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1732095494829199)

File: src/languages/en.ts [hunk 1]
```diff
@@ -3789,6 +3789,10 @@ const translations = {
             xero: 'Xero',
             netsuite: 'NetSuite',
             intacct: 'Sage Intacct',
+            talkYourOnboardingSpecialist: 'Chat with your setup specialist.',
+            talkYourAccountManager: 'Chat with your account manager.',
+            talkToConcierge: 'Chat with Concierge.',
+            needAnotherAccounting: 'Need another software? ',
```
[the remark this answers is not in the document]
Comment (nkdengineer) [reply src/languages/en.ts]: Yes

**Comment by nkdengineer** [comment]
@dukenv0307 i updated, please check again

**Comment by danielrvidal** [comment]
@dukenv0307 how is the review going?

**Comment by dubielzyk-expensify** [comment]
Yeah, it looks a bit off. Here's some more detailed specs to help you out:
CleanShot 2024-11-29 at 10 47 52@2x

Also, the text color on this bit should be `text-supporting` like seen below:
CleanShot 2024-11-29 at 10 48 50@2x

**Review by dukenv0307 — APPROVED** [review]

**Comment by dubielzyk-expensify** [comment]
That looks good to me 👍

**Review by dubielzyk-expensify — DISMISSED** [review]

**Review by techievivek — COMMENTED** [review]
Just posted my concern, let me know if you have any questions, thanks.

**Comment by nkdengineer** [comment]
I'll update soon

**Comment by nkdengineer** [comment]
@dukenv0307 @techievivek i updated to call API `GetAssignedSupportData` but it didn't have data 

Screenshot 2024-12-03 at 15 36 37

**Comment by techievivek** [comment]
@nkdengineer I think that's expected since you don't have a guide/account manger/partner manager assigned to you. Did you try creating a workspace and see if you get assigned to a guide?

**Comment by nkdengineer** [comment]
> @nkdengineer I think that's expected since you don't have a guide/account manger/partner manager assigned to you. Did you try creating a workspace and see if you get assigned to a guide?

https://github.com/user-attachments/assets/3b25ad03-3791-497a-a9b4-597a3c39dd08

@techievivek @dukenv0307 I tried creating a new workspace and still haven't received data. Am I missing something?

**Comment by techievivek** [comment]
@nkdengineer Can you try with a new email that is not alias email format(include `+`), we don't assign guides to account with alias email.

**Comment by nkdengineer** [comment]
Screenshot 2024-12-05 at 15 44 30

thanks @techievivek, It looks like we have data now

**Comment by techievivek** [comment]
@nkdengineer You will be updating the PR to check for the assigned guide before showing the `chat with setup specialist` link, right? Just double checking to see if you understand my comment that I posted earlier.

**Comment by nkdengineer** [comment]
@techievivek @dukenv0307  Sorry, I’m not entirely clear. Do you mean that we will only show the text below if `account.guideDetails` exists?

Screenshot 2024-12-06 at 15 51 09

**Comment by techievivek** [comment]
Yeah, we only want to show the link when `account.guideDetails.email` entry is non-empty.

**Comment by techievivek** [comment]
@nkdengineer Do you need any more clarifications here? Feel free to DM me on Slack if you have any questions, thanks.

**Comment by nkdengineer** [comment]
Thanks @techievivek. I'm clear now. I'll give an update soon

**Comment by nkdengineer** [comment]
@dukenv0307 @techievivek I updated code changes and test steps. Please check again

**Comment by techievivek** [comment]
email and calendarLink should be fine for now, rest are for internal purposes.

**Comment by nkdengineer** [comment]
@dukenv0307 i updated

**Comment by techievivek** [comment]
@dukenv0307 Can you please do a final review here? Thanks.

**Review by dukenv0307 — APPROVED** [review]

**Review by techievivek — APPROVED** [review]

**Comment by danielrvidal** [comment]
Looking great on staging! 
image

**Pull request merged**